napari/plugins/_tests/fixtures/invalid_plugin-0.1.0.dist-info/METADATA

```diff
@@ -2,6 +2,6 @@ Metadata-Version: 2.1
 Name: invalid
 Version: 0.1.0
 Maintainer-Email: example@invalid.com
-Home-Page: https://www.example.com
+Home-Page: https://github.com/napari/napari
```

Comment:
i wanted at least one of them to be a valid github url so that we can test the "submit issue at github" button.  I'll change it to my personal fork so we don't get any accidental issues.